lib/presentation/views/settings/settings_info.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class SettingsInfoView extends StatelessWidget {
+  final String message;
+
+  const SettingsInfoView({Key? key, this.message = '\n \n \n Blank'})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // body: Center(
+      body: Column(
```

Comment:
Wrap the Column into a SingleChildScrollView with added padding
SingleChildScrollView( primary: false, padding: const EdgeInsets.all(AppPaddingValues.smallPadding), child: Column(...

Reply:
done